python/lsst/rubintv/production/starTracker.py

```diff
@@ -399,6 +401,7 @@ def __init__(self,
         self.solver = CommandLineSolver(indexFilePath=self.locationConfig.astrometryNetRefCatPath,
                                         checkInParallel=True,
                                         timeout=30)
+        self.fig = plt.figure(figsize=(16, 16))
```

Comment:
Is there ever a case where you would want to pass an existing figure reference into this class?

Reply:
No that I can think of tbh.